files/test_programs/light0.cpp

```diff
@@ -0,0 +1,24 @@
+#include <fstream>
+#include <fstream>
+#include <string>
+#include <unistd.h>
+using namespace std;
+
+#define LED0_PATH "/sys/class/leds/beaglebone:green:usr0"
+
+int main(int argc, char* argv[]){
+	fstream fs;
+	
+	while(1) {
+		fs.open(LED0_PATH "/brightness", fstream::out);
+		fs << "1";
```

Comment:
This is really the way to set an LED? Very strange compared to the way we do it in embedded

Reply:
This is only the case for the USER LEDs on the board. I do not have a breadboard or LEDs to use the GPIO pins to configure them.